DEVELOP.md

```diff
@@ -0,0 +1,485 @@
+    ├── daffodil-runtime1/
+    │   └── src/               - Contains Daffodil's Scala parser
+    ├── daffodil-runtime1-layers/
+    │   └── src/               - Contains Daffodil's Scala layer transformers
+    ├── daffodil-runtime1-unparser/
+    │   └── src/               - Contains Daffodil's Scala unparser
+    ├── daffodil-runtime2/
+    │   └── src/               - Contains Daffodil's C code generator
+    ├── daffodil-sapi/
+    │   └── src/               - Contains Daffodil's Scala API
+    ├── daffodil-schematron/
+    │   ├── README.md          - Explains how to validate with Schematron rules
+    │   └── src/               - Contains Daffodil's Schematron validator
+    ├── daffodil-tdml-lib/
+    │   └── src/               - Contains Daffodil's TDML runner
+    ├── daffodil-tdml-processor/
+    │   └── src/               - Contains Daffodil's Scala & C TDML implementations
+    ├── daffodil-test/
+    │   └── src/               - Contains Daffodil's TDML tests
+    ├── daffodil-test-ibm1/
+    │   └── src/               - Contains more TDML tests from IBM & Tresys
+    ├── daffodil-udf/
+    │   ├── README.md          - Explains how to implement user defined functions
+    │   └── src/               - Contains Daffodil's UDF interfaces 
+    ├── project/
+    │   ├── Dependencies.scala - Defines third party dependencies
+    │   ├── Rat.scala          - Lists files to be ignored by license audit tool
+    │   ├── build.properties   - Defines which sbt version to use
+    │   └── plugins.sbt        - Adds third party plugins to sbt
+    ├── test-stdLayout/
+    │   └── src/               - Contains TDML tests laid out in different ways
+    └── tutorials/
+        └── src/               - Contains Daffodil's TDML test tutorials
+
+When compiled into a command line interface, Daffodil is composed of a
+script, a log4j2 configuration file, 14 Daffodil jars, and 37
+third-party dependency jars.  These numbers are accurate for Apache
+Daffodil release 3.4.0, but may be different for previous or
+later releases.  Daffodil's command line interface is released in
+binary form as follows:
+
+    apache-daffodil-3.4.0-bin/
+    ├── LICENSE                - Contains licenses of Daffodil and subcomponents
+    ├── NOTICE                 - Contains notices of Daffodil and subcomponents
+    ├── README.md              - Explains how to start Daffodil
+    ├── bin/
+    │   ├── daffodil           - Starts Daffodil on Linux
+    │   └── daffodil.bat       - Starts Daffodil on Windows
+    ├── conf/
+    │   └── log4j2.xml         - Configures log4j logger output
+    └── lib/                   - Contains Daffodil jars and third party jars
+
+When an application uses Daffodil's Java or Scala API (not counting
+the jars used only by Daffodil's command line interface), it will need
+to put 8 Daffodil jars and 18 third-party dependency jars on its
+classpath.  These numbers are accurate for Apache Daffodil release
```

Comment:
Same here, not sure these numbers are needed. It also depens on which jars you need. For example, some projects may not require the tdml-processor if they don't care about testing, or use some other infrastructure for testing.